src/mlpack/core/data/binarize.hpp

```diff
@@ -45,10 +45,10 @@ void Binarize(const arma::Mat<T>& input,
   output.copy_size(input);
 
   const int totalElems = static_cast<int>(input.n_elem);
-  const T *inPtr = input.memptr();
-  T *outPtr = output.memptr();
+  const T* inPtr = input.memptr();
+  T* outPtr = output.memptr();
 
-  #pragma omp parallel for
+#pragma omp parallel for
```

Comment:
There is no option for intending preprocessor separately in `.clang-format` file. It just finds statements with `#` and intends accordingly. One thing we can do is add comments `clang-format off` and `clang-format on` in this particular code section?